tests/core/test_flask_sqlalchemy_session.py

```diff
@@ -0,0 +1,95 @@
+from unittest import mock
+
+import pytest
+from flask import Flask
+from sqlalchemy import create_engine
+from sqlalchemy.orm import Session, sessionmaker
+
+from core.flask_sqlalchemy_session import current_session, flask_scoped_session
+
+
+@pytest.fixture
+def sqlite_engine():
+    """Return an sqlite engine"""
+    return create_engine("sqlite://")
```

Comment:
These are the original tests from the upstream library we pulled in. The old tests used sqllite, so I left them like that, although we could change them to use postgres instead here.